build.rs

```diff
@@ -1,22 +1,47 @@
 use std::{
     env,
-    ffi::OsString,
-    fs,
-    io,
-    io::{ Error, ErrorKind },
-    path::{ Path, PathBuf },
-    process::{ exit, Command },
+    ffi::{OsStr, OsString},
+    fs, io,
+    io::{Error, ErrorKind},
+    path::{Path, PathBuf},
+    process::{exit, Command},
 };
 
 #[cfg(target_family = "unix")]
-use std::{
-    os::unix::ffi::OsStrExt,
-    ffi::OsStr,
-};
+use std::os::unix::ffi::OsStrExt;
 
 #[cfg(target_family = "windows")]
 use std::os::windows::ffi::OsStringExt;
 
+//
+// Environmental variables
+//
+
+// The environmental variables that are usually set by R. These might be needed
+// to set manually if we compile libR-sys outside of an R session.
+//
+// c.f., https://stat.ethz.ch/R-manual/R-devel/library/base/html/EnvVar.html
+const ENVVAR_R_INCLUDE_DIR: &str = "R_INCLUDE_DIR";
+const ENVVAR_R_HOME: &str = "R_HOME";
+
+// An R version (e.g., "4.1.2" or "4.2.0-devel"). When this is set, the actual R
+// binary is not executed. This might be useful in some cases of cross-compile.
+// c.f., https://github.com/extendr/libR-sys/issues/85
+const ENVVAR_R_VERSION: &str = "LIBR_SYS_R_VERSION";
```

Comment:
Can we rename this to `LIBRSYS_R_VERSION` for consistency with other variables, if it's not too late...? Sorry I didn't notice this.

Reply:
Sure.